src/tests/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ foreach(PROG
   test_interaction
   test_lammpsdatareader 
   test_lammpsdumpreaderwriter
+  test_molecule
```

Comment:
There is already a test called `test_molecule` in xtp, so you will need to rename one of the two.

Reply:
Good point any suggestions?

Reply:
`csg_test_molecule` ?